miri

```diff
@@ -156,6 +159,10 @@ run|run-debug)
     # Then run the actual command.
     exec cargo run $CARGO_BUILD_FLAGS -- --sysroot "$MIRI_SYSROOT" "$@"
     ;;
+fmt)
+    find "$MIRIDIR" -not \( -name target -prune \) -name '*.rs' \
+        | xargs rustfmt --edition=2021 --config-path "$MIRIDIR/rustfmt.toml" "$@"
```

Comment:
Notably this `"$@"` supports `./miri fmt --check` which I plan to use in CI.

Reply:
Why does this PR not use it on CI then?